app/uk/gov/hmrc/mobilepayments/connectors/ShutteringConnector.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.mobilepayments.connectors
+
+import com.google.inject.name.Named
+import com.google.inject.{Inject, Singleton}
+import play.api.Logger
+import play.api.libs.json.JsValue
+import uk.gov.hmrc.http.HttpReads.Implicits._
+import uk.gov.hmrc.http.{CoreGet, HeaderCarrier, Upstream5xxResponse}
+import uk.gov.hmrc.mobilepayments.domain.Shuttering
+import uk.gov.hmrc.mobilepayments.domain.types.ModelTypes.JourneyId
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class ShutteringConnector @Inject()(
+  http:                                   CoreGet,
+  @Named("mobile-shuttering") serviceUrl: String) {
+
+  val logger: Logger = Logger(this.getClass)
+
+  def getShutteringStatus(
+    journeyId:              JourneyId
+  )(implicit headerCarrier: HeaderCarrier,
+    ex:                     ExecutionContext
+  ): Future[Shuttering] =
+    http.GET[JsValue](s"$serviceUrl/mobile-shuttering/service/mobile-self-assessment/shuttered-status?journeyId=$journeyId").map {
```

Comment:
copy/paste typo here "mobile-self-assessment"

I'd suggest going with mobile-open-banking as the service name to give us a bit of flexibility in how we shutter the different parts of this service.

Reply:
Ah yes - copy and paste error there. Cheers